spec/depicts/misc/camera.yaml

```diff
@@ -8,4 +8,5 @@ exclude:
   - Category:Camera parts
   - Category:Camera showrooms
   - Category:Camera accessories
-excludeRegex: "/(in art)/i"
+  - Category:Photographs by camera manufacturer
+excludeRegex: "/(in art|Taken with)/"
```

Comment:
should add the `/i` back so that it is case insensitive.

Reply:
It didn't feel necessary in this case, but sure, it's not likely that capitalization variations in these patterns would represent different intended meanings. I'll add it back.